docs/content/how-to/aws/disaster-recovery.md

```diff
@@ -1,17 +1,22 @@
 ---
-title: Migrating Hosted Cluster among the same AWS Region
+title: Disaster Recovery
 ---
 
-# Migrating Hosted Cluster among the same AWS Region
+# Disaster Recovery
 
-## Preface and Considerations
+## Migrating Hosted Cluster among the same AWS Region
+
+### Preface and Considerations
 
 The behaviour of this implementation is focused on the transparency for the user. Hypershift will not disrupt any customer workloads at anytime, also have in mind that the service workloads will be up and running during the migration process. Maybe at some point the Cluster API will be down but this will not affect the services running on the worker nodes.
 
 In the storage side, It's mandatory to have under consideration that when we move a HostedControlPlane to another Management cluster we use some external services, which allows the migration to make it happen. We re-use the storage provisioned in AWS by the initial ControlPlane (PVs/PVCs) in the destination Management cluster.
 
 Regarding the Workers nodes assigned to the cluster, during the migration they will still point to the same DNS entry, and we, under the hood, change the DNS Records to point to the new Management Cluster API, that way the node migration is transparent for the user.
 
+!!! important
+    Have in mind that this **"migration"** capability, it's only for **disaster recovery purposes**, please **DO NOT** use this to perform clusters migrations as a common task in your platform.
```

Comment:
As discussed, let's try to add an example of what disaster recovery is, and describe it at the top of the page given it's a new topic.

Reply:
Done